src/style.css

```diff
@@ -1 +1,77 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+  font-style: normal;
+}
+
+.page {
+  color: #3d4e61;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+.page__form {
+  margin-top: 20px;
+}
+
+.page__body {
+  margin-block: 0;
+  display: flex;
+  flex-direction: column;
```

Comment:
Don't fix container size (if there is no such a requirement). In this case, you're not fixing the size, but make sure the content dictates the size of the container.